packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -39,69 +39,66 @@ import { PersistencePromise } from './persistence_promise';
 import { QueryCache } from './query_cache';
 import { QueryData } from './query_data';
 import { SimpleDbStore, SimpleDbTransaction } from './simple_db';
+import { TargetIdGenerator } from '../core/target_id_generator';
 
 export class IndexedDbQueryCache implements QueryCache {
   constructor(private serializer: LocalSerializer) {}
 
-  /**
-   * The last received snapshot version. We store this seperately from the
-   * metadata to avoid the extra conversion to/from DbTimestamp.
-   */
-  private lastRemoteSnapshotVersion = SnapshotVersion.MIN;
-
-  /**
-   * A cached copy of the metadata for the query cache.
-   */
-  private metadata = null;
-
```

Comment:
Can you add a comment something like:

PORTING NOTE: We don't cache global metadata for the query cache, since some of it (in particular highestTargetId) can be modified by secondary tabs. We could perhaps be more granular (and e.g. still cache lastRemoteSnapshotVersion in memory) but for simplicity we currently go to IndexedDb whenever we need to read metadata. We can revisit if it turns out to have a meaningful perf impact.

Reply:
Added this comment. Thanks!